frontend/src/components/navbar/top-navbar.tsx

```diff
@@ -1,84 +1,142 @@
+        color: 'white',
     },
-})
-
-const AdminLinks = loadAsync('Admin Links', () => import('./admin-links'))
-const AccountLinks = loadAsync('Account Links', () => import('./account-links'))
+    '&.active::after': {
+        content: '""',
+        position: 'absolute',
+        bottom: '-2px',
+        left: 0,
+        width: '100%',
+        height: '4px',
+        backgroundColor: '#0EE094',
+    },
+});
 
-export const TopNavBar: FCWOC<TopNavBarProps> = ({ children, className }) => {
-    const user = useCurrentUser()
-    const hideBanner = user.isResearcher
+const AdminLinks = loadAsync('Admin Links', () => import('./admin-links'));
+const AccountLinks = loadAsync('Account Links', () => import('./account-links'));
+export const TopNavBar: React.FC<TopNavBarProps> = ({ className }) => {
+    const user = useCurrentUser();
+    const hideBanner = user.isResearcher;
+    const isMobile = useIsMobileDevice();
 
     return (
-        <nav className={cx('navbar', 'navbar-light', className)}>
-            <div className="navbar-dark bg-dark py-1">
-                <Container>
-                    <Box justify="between" align="center" gap padding={{ vertical: 'default' }}>
-                        <NavbarLogoLink />
-                        {children}
-                        <Box gap="xlarge" align="center">
-                            <Link to="/studies" css={{ color: 'white', textDecoration: 'none', fontWeight: 'bold' }}>
-                                Dashboard
-                            </Link>
-                            <DesktopResearcherLinks />
-
+        <nav className={cx('navbar', 'navbar-light', className)} style={{ backgroundColor: colors.navy, color: 'white', height: '80px' }}>
+            <div style={{ height: '80px', backgroundColor: colors.navy }}>
+                <Container style={{ height: '80px' }}>
+                    <Group 
+                        justify="space-between"
+                        align="center"
+                        style={{
+                            width: '100%',
+                            height: '80px',
+                        }}
+                    >
+                        <Group>
+                            <NavbarLogoLink /> 
+                        </Group>
+                        <Group
+                            justify="center"
+                            align="center"
+                            gap="40px"
+                            style={{
+                                height: '80px',
+                                padding: '26px 0px 26px 0px',
+                            }}
+                        >
+                            {!isMobile && !user.isAdministrator && !user.isResearcher && (
+                                <>
+                                    <Wrapper>
+                                        <NavbarStyledLink to="/studies" className={({ isActive }) => isActive ? 'active' : ''}>
+                            Dashboard
```

Comment:
Should be named as All Studies, the active style is not per design spec, it is more thick and hugging the letters, should be spaced out.

Reply:
The navbarstyled links are not displayed on mobile